packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -1093,6 +1099,7 @@ export class ProjectModel {
             await copyChartVersionContent(
                 'saved_queries_version_additional_metrics',
                 ['saved_queries_version_additional_metric_id', 'uuid'],
+                { filters: (value: any) => JSON.stringify(value) },
```

Comment:
if this can be unknown that will be great! same thing in the fieldPreprocess arg. we've been victims of `any` too many times now 😆

Reply:
btw, just curious, any ideas why the json parser was failing?